ApollosProject/apollos-prototype
# Adds Content URL Sharing

## DESCRIPTION

### What does this PR do, or why is it needed?

Adds a sharing URL to content items. If the content channel has no `itemUrl`, it just returns the top level domain.

### What design trade-offs/decisions were made?

Not really a trade-off because I don't think there's any other way, but this requires the content channel to have something in the `ItemUrl` attribute. And it's not automatic like the content item slugs are (as far as I know)

### How do I test this PR?

Pull down the `userFeed` and include `sharing.url` in the query. I've only pre-filled the news channel with a url, if you wanna see more, you can add them.

### What automated tests did you write?

One data source test

## TODO:

- [x] Closes [#796]

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
